backend/dataall/aws/handlers/glue.py

```diff
@@ -370,142 +368,6 @@ def batch_delete_tables(**data):
             )
             raise e
 
-    @staticmethod
-    @Worker.handler(path='glue.dataset.crawler.create')
-    def create_crawler(engine, task: models.Task):
-        with engine.scoped_session() as session:
-            dataset: models.Dataset = db.api.Dataset.get_dataset_by_uri(
-                session, task.targetUri
-            )
-            location = task.payload.get('location')
-            Glue.create_glue_crawler(
-                **{
-                    'crawler_name': f'{dataset.GlueDatabaseName}-{location}'[:52],
-                    'region': dataset.region,
-                    'accountid': dataset.AwsAccountId,
-                    'database': dataset.GlueDatabaseName,
-                    'location': location or f's3://{dataset.S3BucketName}',
-                }
-            )
-
-    @staticmethod
-    def create_glue_crawler(**data):
```

Comment:
What happened with this method? I do not see it in glue_dataset_client, was it unused?

Reply:
This was the dead code. It should have been been invoked by sending SQS action `glue.dataset.crawler.create`, but there was no `glue.dataset.crawler.create` anymore in the project, I deleted it